scripts/bulk-inserter.js

```diff
@@ -0,0 +1,63 @@
+// This could also be done as a shell script, but that would not allow windiws devs to execute it
+// Args:
+// 1. localhost port (defaults to 3002. To see why this script only works against localhost see README)
+// 2. filepath (defaults to sample data)
+// 3. chunksize (defaults all at once)
+
+const { readFileSync, existsSync } = require("fs");
+
+function readJSON(filepath) {
+  if (!existsSync(filepath)) {
+    throw new Error("File could not be found");
+  }
+  const rawData = readFileSync(filepath).toString('utf-8');
+  return JSON.parse(rawData)
+}
+
+function splitIntoChunks(data, chunkSize) {
+  const count = 0;
+  const res = [];
+  while (count < data.length) {
+    res.concat(data.slice(count, count + chunkSize));
+    count += chunkSize;
+  }
+  return res;
+}
+
+async function sendData(port, data) {
+  console.log(`http://localhost:${port}/commands/v1/items/bulk-insert`)
+  const res = await fetch(`http://localhost:${port}/commands/v1/items/bulk-insert`, {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(data),
+  });
+  // Is reponse ok?
+  if (res.status !== 200) {
+    throw new Error(`Request failed with statuscode ${res.status}`);
+  }
+}
+
+async function main(opts) {
+  const data = readJSON(opts.filePath);
+  console.log(`Found json with ${data.length} entries`);
+  let chunks = [data];
+  if (opts.chunkSize !== -1) {
+    chunks = splitIntoChunks(data, opts.chunkSize);
+  }
+  let count = 0;
+  for (const chunk of chunks) {
+    console.log(`Inserting chunk ${count}/${chunks.length}`);
+    await sendData(opts.port, chunk);
+  }
+  console.log("Data inserted!");
+}
+
+const opts = {
+  port: +process.argv.at(2) || 3002,
+  filePath: process.argv.at(3) || "./data/sample.json",
+  chunkSize: process.argv.at(4) || -1,
+};
+
+main(opts);
```

Comment:
This looks very good! The chunk part is very efficient for large datasets.

I would ensure to use typescript here as well. Either by using `ts-node` or "compiling" the script and running the "compiled" version. We should always use typescript. 

We do not add this built in to the command backend because of the option to insert the data into another system (like production), isn't it? Because I think a solution inside the command backend (while in DEV) would be a good place to put this.

Reply:
I can see what you mean, yes. However using typescript requires the overhead of a package.json as far as I can tell and I would like for this folder to have as little clutter as possible.

I would advise against bundling this into the docker image. It is not a part of the application itself and could be considered bloat. In addition to that it may give a misleading representation of the application functionality when the db fills itself magically at startup

Reply:
Good points. I totally agree to not seed the db magically. 

However I think `typescript > "overload/clutter"` + we should test the script. 

So if we move the script into the backend while in development. We can use the test system and typescript without any additional setup. Also if types or endpoints change we would instantly see these changes (failing tests, types are incorrect). 

I think we should do this because these benefits are better than to keep it out of the docker itself.

Reply:
Fair point, however I would prefer to use the script folder as it is a place where dev scripts are stored. And this is nothing but a dev script 🤔 
Also it would be the intuitive place to look when trying to bulk insert data into production.

Reply:
Also when starting the docker compose and trying to populate it, adding the script in a root level folder and not somewhere in multiple nested folders makes it more approachable imo

Reply:
>Fair point, however I would prefer to use the script folder as it is a place where dev scripts are stored. And this is nothing but a dev script 🤔
Also it would be the intuitive place to look when trying to bulk insert data into production.

I understand this but I still think the benefits (when also considering UX of devops) are worth to move it into the command backend. 

>Also when starting the docker compose and trying to populate it, adding the script in a root level folder and not somewhere in multiple nested folders makes it more approachable imo

Oh what I mean is to include the logic into the backend which (in development) than has an endpoint /seed-data or something like this.